src/Compilers/CSharp/Test/Syntax/Parsing/CSharpParseOptionsTests.cs

```diff
@@ -21,6 +22,18 @@ private void TestProperty<T>(Func<CSharpParseOptions, T, CSharpParseOptions> fac
             Assert.Same(newOpt2, newOpt1);
         }
 
+        [Fact]
+        public void LanguageVersionFactsCurrentVersion()
```

Comment:
>LanguageVersionFactsCurrentVersion [](start = 20, length = 34)

Strange test: My understanding that it verifies that the current version is the previous version before the latest.

Will we always have this assumption or is this an assumption between C#7 and C#8?

Reply:
No, it validates that the current version is the max value, ignoring latest since that's always `int.MaxValue`.